manifests/b/Beeper/Beeper/4.0.551/Beeper.Beeper.installer.yaml

```
@@ -0,0 +1,25 @@
+# Created with komac v2.11.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Beeper.Beeper
+PackageVersion: 4.0.551
+InstallerLocale: en-US

```

```suggestion
InstallerLocale: en-US
```

Adding InstallerLocale may cause updating issues.